packages/js/src/workouts/components/ConfigurationWorkout.js

```diff
@@ -591,7 +601,7 @@ export function ConfigurationWorkout( { toggleStep, toggleWorkout, isStepFinishe
 						finishText={ "Finish this workout" }
 						onFinishClick={ toggleConfigurationWorkout }
 						isFinished={ isStepFinished( "configuration", steps.newsletterSignup ) }
-						additionalButtonProps={ { disabled: indexingState !== "completed" } }
+						additionalButtonProps={ { disabled: ( ( indexingState !== "completed" ) || ( state.tracking !== 0 && state.tracking !== 1 ) ) } }
```

Comment:
Maybe we should make this a bit more readable? (also thinking what could happen if we add conditions)